CAKE-autorate.sh

```diff
@@ -24,13 +33,37 @@ cleanup_and_killall()
 	exit
 }
 
+# check to see if the named interface is present; exit if not
+is_if_present()
+{
+    the_if=$1
+    is_here=$(ifconfig | grep "$the_if")
+    if [ "$is_here" == "" ]; then
+        log_msg "'$the_if' interface not present... Aborting!\n"
+		log_msg "Install SQM QoS and configure CAKE-config.sh\n"
```

Comment:
```suggestion
		
```

Reply:
If the interface doesn't exist that is not necessarily because SQM QoS needs to be installed, right? But the line above seems fine.

Reply:
I'll change it to say, "Is SQM QoS intalled and CAKE-autorate.sh updated?\n"